test/stubs/contentful/get-entries-campaign-with-overrides.json

```diff
@@ -0,0 +1,40 @@
+{
+  "sys": {
+    "type": "Array"
+  },
+  "total": 1,
+  "skip": 0,
+  "limit": 100,
+  "items": [
+    {
+      "sys": {
+        "space": {
+          "sys": {
+            "type": "Link",
+            "linkType": "Space",
+            "id": "owik07lyerdj"
+          }
+        },
+        "id": "6tlsrM2RTa40oMqwcIKSg6",
+        "type": "Entry",
+        "createdAt": "2017-02-15T20:30:31.426Z",
+        "updatedAt": "2017-05-12T15:20:14.634Z",
+        "revision": 10,
+        "contentType": {
+          "sys": {
+            "type": "Link",
+            "linkType": "ContentType",
+            "id": "campaign"
+          }
+        },
+        "locale": "en-US"
+      },
+      "fields": {
+        "campaignId": "7",
+        "gambitSignupMenuMessage": "Great! Little moments can have a big impact. Complimenting someone can turn their whole day around.\n\nDo this: post encouraging notes in places that can trigger low self-esteem, like school bathrooms.\n\nThen, text START to share a photo of the messages you posted (and you'll be entered to win a $1500 scholarship)! \n\nBonus: Text back a photo of your posted note by 11:59PM tonight and you can enter to win a gift card of your choice! (Winners announced this week). ",
```

Comment:
@aaronschachter 
No test should be testing the data itself, just the existence of properties, and states of being empty or not, but not actual string comparisons, because of this same reason. The text you refer to is the actual value in contenful for this override, the `text START` is hardcoded as the string returned in this field, in contentful. This, is in my opinion, is a potential source for a bug since it could run out of sync with what we set in the env variable (although we rarely change this), but this is a different issue.

Any JSON stub I use, I think of it as stale data, so I don't test if the stubs particular string values are correct, but rather that the data structure is what I expect when stubbing functions and getting back the created objects, etc. I also use them to force the unit tests to follow a certain branch of the method, then I test for expected methods being called in that particular function branch.

Creating cyborg stubs really doesn't solve the problem that the actual values in them can and will run out of sync. If we just make them dynamic, we are going to run the risk of treating them like actual important sources of truth, relying on them and the fake sense of trust that will give us (sorry stubs, not personal). I think a better solution, for the future, would be to create a simple module that could update this stubs with more "fresh" JSON data. This way, every few weeks (or whatever strategy we devise), we could run it and end up with fresh JSON data that reflects any potential changes in the data structures of objects being returned by our platform.

My 2 cents.